craig-rolleston/week2/fri/ga-bank/js/atm.js

```diff
@@ -0,0 +1,179 @@
+
+// console.log('Hello ATM challenge');
+const atm = function(){
+
+
+    const $div = $('div');
+    const $divCheckingAccount = $('#checking.account');
+    const $divSavingsAccount = $('#savings.account');
+    const $body = $('body');
+    const $input = $('input');
+    const $balance = $('.balance');
+        
+    $div.css({
+        width: '100%',
+        marginTop: '20px',
+    }),
+
+    $divCheckingAccount.css({
+        borderRadius: '5px',
+        padding: '10px',
+        width: '40%',
+        marginLeft: '65px',
+        marginRight: '5px',
+
+    }),
+
+
+    $divSavingsAccount.css({
+        borderRadius: '5px',
+        padding: '10px',
+        width: '40%',
+        marginRight: '65px',
+        marginLeft: '5px',
+
+    }),
+
+
+    $body.css({
+        background: 'black',
+
+    }),
+
+
+    $input.css({
+        borderRadius: '5px',
+        padding: '5px',
+        marginTop: '5px',
+
+    })
+
+    $balance.css({
+        borderRadius: '5px',
+        backgroundColor: 'rgb(211, 211, 211)',
+        marginBottom: '5px',
+    })
+
+    // debugger
+
+
+    // ATM
+
+
+    // CHECKING DEPOSIT
+    let checkingBalance = 0; // base value 
+    const $displayCheckingBalance = $('#checking-balance');
+
+    const checkingDeposit = function() {
+
+        let checkingValue = parseInt($(`#checking-amount`).val());
+        checkingBalance = checkingBalance + checkingValue;
```

Comment:
What if i gave you a negative amount? 
i.e. `checking balance is $100` and `checkingvalue is -50`?
the new balance will be $50

Reply:
what if i entered nothing?